packages/medusa/src/services/totals.ts

```diff
@@ -208,14 +208,7 @@ class TotalsService extends TransactionBaseService {
     }
 
     if (opts.include_tax) {
-      if (isOrder(cartOrOrder) && cartOrOrder.tax_rate !== null) {
-        totals.original_tax_total = Math.round(
-          totals.original_tax_total * (cartOrOrder.tax_rate / 100)
```

Comment:
@srindom can you confirm that it was a bug and it should not be original_tax_total * tax_rate but instead price * tax_rate?

Reply:
Yes you are right!

Reply:
Thanks, in that case it answer my comment here https://github.com/medusajs/medusa/pull/2130#discussion_r961804701 👌